README.md

```diff
@@ -23,14 +26,27 @@ Wireless Tool for ITC
 ### File Structure
 
 ```
-├──[wireless-tool]/              # root directory
-|  └──[README.md]                # this file!
+├──[wireless-tool]/ # root directory
+    ├── README.md
+    ├── requirements.txt
+    ├── server
+    │   ├── components
+    │   ├── config
+    │   └── index.py # the root file of the server
+    ├── bin        
+    ├── include
+    └── lib
 ```
 
 For more detailed documentation on our file structure and specific functions in the code, feel free to check the project files themselves.
 
 ## Setup Steps 
-1. Clone repo by running `git clone https://github.com/dali-lab/wireless-tool` in your terminal
+1. Clone the dev branch `git clone https://github.com/dali-lab/wireless-tool.git --branch dev` in your terminal
```

Comment:
we should probably instruct people to install the `master` branch, `dev` is for internal use. (this pull request, for instance, is merging into `master`)